CHANGELOG.txt

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+v4.1.0
+------
+* Fix: update price and informations on Alma payment method when shipping price change
+* Fix: display right Alma payment method for the current basket price
+* Fix: manage events on next step button in checkout
+* Fix: update custom data in Alma payload
```

Comment:
do you have check custom data payload ? ( null )

Reply:
Yes and it's ok